ext/gettext/gettext.c

```diff
@@ -78,12 +78,13 @@ PHP_MINFO_FUNCTION(php_gettext)
 /* {{{ Set the textdomain to "domain". Returns the current domain */
 PHP_FUNCTION(textdomain)
 {
-	char *domain_name = NULL, *retval;
+	char *retval = NULL, *domain_name = NULL;
 	zend_string *domain = NULL;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "S!", &domain) == FAILURE) {
-		RETURN_THROWS();
-	}
+	ZEND_PARSE_PARAMETERS_START(0, 1)
```

Comment:
Did you intentionally want to change ZPP to the new one? Is it for performance?

Reply:
Yes and no. Just for modernisation sake.

Reply:
I also like fast ZPP, but AFAIK we should carefully use it because it increases code side due to inlining. I don't know what kind of impact it has on ext/gettext ¯\_(ツ)_/¯

Reply:
Apparently it's negligible, as someone benchmarked it last time I complained, especially for small functions which don't have many args.

Reply:
thanks for the info! :)